typescript/lib/constructor_calls.ts

```diff
@@ -31,12 +36,184 @@ export function newGetFileGenerationOptions(filesToGenerateSet: Set<string>): Ma
 }
 
 export function newGetLoadStaticFilesArgs(serviceId: ServiceID, staticFilesToCopyStringSet: Map<String, boolean>): LoadStaticFilesArgs {
-    const result = new LoadStaticFilesArgs();
+    const result: LoadStaticFilesArgs = new LoadStaticFilesArgs();
```

Comment:
I love that you're explicitly typing things here. Explicit is always better than implicit